src/components/Modal/VaultStrategyModal/managedVaults/ManagedVaultsStrategyModalAllocations.tsx

```diff
@@ -0,0 +1,181 @@
+import { ArcElement, Chart as ChartJS, Colors, Legend, Tooltip } from "chart.js"
+import { FC } from "react"
+import { Doughnut } from "react-chartjs-2"
+import { Address } from "wagmi"
+
+import { formatPercentage } from "@/lib/helpers"
+
+import { AssetLogo } from "@/components/Asset"
+
+ChartJS.register(ArcElement, Tooltip, Legend, Colors)
+
+export const ManagedVaultsStrategyModalAllocations: FC = () => {
+  const addresses: Address[] = [
+    "0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9",
+    "0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8",
+    "0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1",
+    "0x82aF49447D8a07e3bd95BD0d56f35241523fBab1",
+    "0x11cDb42B0EB46D95f990BeDD4695A6e3fA034978",
+    "0x17FC002b466eEc40DaE837Fc4bE5c67993ddBd6F",
+  ]
+  const data = {
+    labels: ["Red", "Blue", "Yellow", "Green", "Purple", "Orange"],
+    datasets: [
+      {
+        label: "# of Votes",
+        data: [12, 19, 3, 5, 2, 3],
+        backgroundColor: [
+          "rgba(255,99,132,0.2)",
+          "rgba(54,162,235,0.2)",
+          "rgba(255,206,86,0.2)",
+          "rgba(75,192,192,0.2)",
+          "rgba(153,102,255,0.2)",
+          "rgba(255,159,64,0.2)",
+        ],
+        borderColor: [
+          "rgba(255,99,132,1)",
+          "rgba(54,162,235,1)",
+          "rgba(255,206,86,1)",
+          "rgba(75,192,192,1)",
+          "rgba(153,102,255,1)",
+          "rgba(255,159,64,1)",
+        ],
+        borderWidth: 1,
+      },
+    ],
+  }
+
+  const options = {
+    plugins: {
+      legend: {
+        display: false,
+      },
+      tooltip: {
+        enabled: false,
+        external: externalTooltipHandler,
+      },
+    },
+  }
+
+  const sumTotal = data.datasets?.[0]?.data.reduce(
+    (accumulator, currentValue) => accumulator + currentValue,
+    0
+  )
+
+  return (
+    <div className="grid w-full grid-cols-1 grid-rows-[auto,1fr]">
+      <div>
+        <Doughnut data={data} options={options} className="w-full" />
+        {/* custom tooltip */}
+        <div>
```

Comment:
Wonder if this div is necessary?

Reply:
Yes, the way I browse the tooltip in the cursor event needs this div.